src/AppCommon/Infra/SerializationOptions.cs

```diff
@@ -9,14 +9,14 @@ public static class SerializationOptions
         public static readonly JsonSerializerOptions IndentedWithNoEscaping = new()
         {
             WriteIndented = true,
-            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+            DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault,
```

Comment:
Additional thought on this class: given this is highly coupled to the report, it may make more sense to move this class to Particular.EndpointThroughputCounter.Data, make it internal, and then move the serialization logic there as well.

Then Particular.EndpointThroughputCounter.Data becomes a package shared by ServiceControl and ParticularTools that has the serialization and validation (obviously validation only works if you have the private key) baked in.

Reply:
Yes the serialization bits look like they would be best suited to be in one place. We still have to figure out how all that will work, ie where the best place for it is - so I don't want to do that in this PR.